doc/control.rst

```diff
@@ -78,8 +79,9 @@ Control system analysis
 
     dcgain
     describing_function
-    evalfr
-    freqresp
+    frequency_response
+    TransferFunction.__call__
+    StateSpace.__call__
```

Comment:
I agree this format is a bit long and cumbersome.  Would it make sense to have this show up as something more like `sys(x[, squeeze])` or something like that.  This can be done by writing the desired signature as the first line of the docstring, as described [here](https://numpydoc.readthedocs.io/en/latest/format.html).

Reply:
Good idea. I tried to rename the call string for `TransferFunction.__call__` in the docstring as suggested in  numpydoc but no luck: 

<img width="815" alt="image" src="https://user-images.githubusercontent.com/58706249/207111337-a5440116-eb75-4aaf-8531-a7c95d659b29.png">

Will see if I can find any examples in numpy doc itself I can use

Reply:
Ok looked around in numpy and scipy and found `scipy.interpolate.interp1d`, which is a class that has a `__call__` method. But it is not linked in a function reference like this, so it did not provide a way to change how the call signature is shown. Perhaps there is a way to tell sphinx to manually change its call signature, but I coudln't find it. 

Unless anybody else has other ideas about how to refer to this, I would propose to leave it as it is in this PR and plan to update later if somebody figures something out.